include/deal.II/base/mpi.h

```diff
@@ -238,6 +238,58 @@ namespace Utilities
               std::vector<T> &maxima);
 
     /**
+     * Return the minimum over all processors of the value @p t. This function
+     * is collective over all processors given in the communicator. If deal.II
+     * is not configured for use of MPI, this function simply returns the
+     * value of @p t. This function corresponds to the
+     * <code>MPI_Allreduce</code> function, i.e. all processors receive the
+     * result of this operation.
+     *
+     * @note Sometimes, not all processors need a results and in that case one
+     * would call the <code>MPI_Reduce</code> function instead of the
+     * <code>MPI_Allreduce</code> function. The latter is at most twice as
+     * expensive, so if you are concerned about performance, it may be
+     * worthwhile investigating whether your algorithm indeed needs the result
+     * everywhere or whether you could get away with calling the current
+     * function and getting the result everywhere.
+     *
```

Comment:
the last part of the sentence `and getting the result everywhere` does not make sense to me.